packages/react-dev-utils/launchEditor.js

```diff
@@ -257,6 +257,11 @@ function launchEditor(fileName, lineNumber) {
   }
 
   let [editor, ...args] = guessEditor();
+
+  if (editor.toLowerCase() === 'none') {
```

Comment:
This could be null.

Reply:
it's ok if i just modify the condition? `editor && editor.toLowerCase() === 'none'`

Reply:
We could just move this condition after the next one.